2 - RTB/Documentazione interna/Verbali/24-01-17/24-01-17.typ

```diff
@@ -0,0 +1,59 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "17/01/24",
+  subTitle: "Meeting di pianificazione RTB",
+  docType: "verbale",
+  authors: (
+    "Todesco Mattia",
+  ),
+  timeStart: "10:00",
+  timeEnd: "11:00",
+);
+
+= Ordine del giorno
+
+- Valutazione progresso generale;
+- verifica avanzamenti per colloquio RTB;
```

Comment:
La prima parola degli elementi dell'elenco puntato va in maiuscolo, in coerenza con gli altri verbali